windows-server-container-tools/Update-ContainerHost/Update-ContainerHost.ps1

```diff
@@ -338,6 +341,114 @@ Find-DockerImages
 
 
 function 
+Install-Docker()
+{
+    [CmdletBinding()]
+    param(
+        [string]
+        [ValidateNotNullOrEmpty()]
+        $DockerPath = "https://aka.ms/tp5/docker"
```

Comment:
https://aka.ms/tp5/docker currently redirects to https://master.dockerproject.org/windows/amd64/docker.exe

This is a moving target as this is the nightly build of Docker.
And AFAIK this is only the Docker Client, or does this also contain the Docker Engine?

But I'm sure this will be fixed until TP5 release.